### DISCUSSION TITLE
dependencies

### DISCUSSION THREAD

**Discussion by t794104** [chocolatey/choco#2455] [Q&A]
In the Choco doc (https://docs.chocolatey.org/en-us/create/create-packages) it says it follows http://docs.nuget.org/docs/reference/versioning. But when i try too use :
    <dependencies>
      <dependency id="xyz" version="4.7.*" />
    </dependencies>

it say's : '4.7.*' is not a valid version string.

So * is not allowed ? the docu suggests otherwise

**Comment by pauby** [comment]
Can you point to where in the docs it says 'it follows the Nuget versioning'? Chocolatey doesn't support '*' in the version.

**Comment by t794104** [comment]
In https://docs.chocolatey.org/en-us/create/create-packages#nuspec at the NuSpec part; it talks about dependencies (chocolatey in this case) and then that part ends with a ref too the NuSpec Version reference. So it does not say it follows... it imply that it follows the spec.

same go's for "Chocolatey includes additional optional functionality on top of NuGet's Nuspec format" (second paragraf of 'Nuspec' chapter. It is based on; so it should follow the specs it then refference ?

PS: got it working now without the *. it's just that the docu is unclean about which parts of the NuSpec specs they follow.

**Reply by pauby** [reply]
Thanks for passing that over. What I see is 'Therefore packages are based on the same principles.' so it's _based_ on it. But it doesn't say explicitly what it supports and doesn't support. So, I complete agree it's not clear. Apologies, you had to jump through some hoops. I think we need to be much clearer on this. I'll go and create an issue for it.

**Reply by pauby** [reply]
I've raised an [issue](https://github.com/chocolatey/docs/issues/289) for this on our docs repository.
